doc/src/modules/physics/vector/fields.rst

```diff
@@ -229,7 +229,7 @@ denoted by :math:`\nabla f` is given by -
 \frac{\partial f}{\partial y}  \mathbf{\hat{j}} +
 \frac{\partial f}{\partial z} \mathbf{\hat{k}}`
 
-To compute the divergence of a vector field in :mod:`physics.vector`, you
+To compute the divergence of a scalar field in :mod:`physics.vector`, you
```

Comment:
I think this should say "gradient of a scalar field".  And a few lines above on ln 225 as well.